src/main/Tree.razor

```diff
@@ -18,6 +18,28 @@
 @using neurUL.Common.Http
 @using System.Linq
 
+@if (this.IsExpandModalVisible)
```

Comment:
Modularize this into a "ProcessBox" similar to "ConfirmationBox" so that Modals for running processes can be reused.